packages/terra-tabs/tests/jest/Tab.test.jsx

```diff
@@ -3,6 +3,8 @@ import React from 'react';
 import { shallowWithIntl } from 'terra-enzyme-intl';
 import Tab from '../../src/common-tabs/_Tab';
 
+jest.mock('uuid', () => ({ v4: () => '00000000-0000-0000-0000-000000000000' }));
```

Comment:
Let's utilize `beforeAll` and `afterAll` blocks e.g.

https://github.com/cerner/terra-framework/blob/698d1309f6a95f2f1737477138809c2bf6985a8f/packages/terra-menu/tests/jest/Menu.test.jsx#L11-L17

Reply:
We should also have unit tests around UUID to make sure that the correct value is being set